supertokens_python/recipe/session/framework/fastapi/__init__.py

```diff
@@ -32,12 +33,15 @@ def verify_session(
     ] = None,
     user_context: Union[None, Dict[str, Any]] = None,
 ) -> Callable[..., Coroutine[Any, Any, Union[SessionContainer, None]]]:
-    if user_context is None:
-        user_context = {}
+    _ = user_context
     from fastapi import Request
 
     async def func(request: Request) -> Union[SessionContainer, None]:
+        nonlocal user_context
         baseRequest = FastApiRequest(request)
+        if user_context is None:
+            user_context = default_user_context(baseRequest)
```

Comment:
this is wrong. We want to modify the user_context even if it's passed by the user to add the default stuff into their passed in user_context.

Reply:
Updated.